src/element.ts

```diff
@@ -18,6 +18,7 @@ export interface ElementAttributes {
   // eslint-disable-next-line
   [name: string]: any;
   id: string;
+  el?: SVGElement;
```

Comment:
Let's make this `any`. I'm sure that we use this for more than just `SVGElement`.

Reply:
done